lib/features/news/screens/news_detail_screen.dart

```diff
@@ -20,7 +20,7 @@ class NewsDetailScreen extends StatelessWidget {
     final theme = Theme.of(context);
     return FutureLoadingScreen<NewsModel?>(
       load: () => fetchNewsById(newsId),
-      buildChild: (NewsModel? news) {
+      buildChild: (NewsModel? news, _) {
```

Comment:
:thinking: This is kinda annoying, but I also did not find a way to avoid this :/